src/you_get/extractors/mgtv.py

```diff
@@ -75,8 +75,9 @@ def prepare(self, **kwargs):
                 size_this_stream = 0
                 stream_fileid_list = []
                 for i in segment_list_this:
-                    _, container_this_stream, size_this_seg = url_info(i)
-                    size_this_stream += size_this_seg
+                    if 'json_output' not in kwargs and not kwargs['json_output']:
```

Comment:
If `'json_output' not in kwargs`, trying to evaluate `not kwargs['json_output']` will raise an exception.
So this bool expression doesn't literally make sense.

@cnbeining Is it possible that `url_info` could take a long time so this PR is trying to get rid of that when retrieving the `--json` output?